package.json

```diff
@@ -54,6 +54,7 @@
     "nuxt-icon": "^0.6.10",
     "ohash": "^1.1.3",
     "pathe": "^1.1.2",
+    "valibot": "^0.31.0",
```

Comment:
Valibot should be an opt-in dependency to promote flexibility in validation library selection and minimize dependencies. It should stay as a dev dependency.